src/org/zaproxy/zap/extension/httpsessions/ExtensionHttpSessions.java

```diff
@@ -258,6 +265,13 @@ private PopupMenuItemCopySessionToken getPopupMenuItemCopySessionToken() {
 		return popupMenuItemCopySessionToken;
 	}
 
+	private PopupMenuSessionSearch getPopupMenuItemFindRelatedMessages() {
+		if (popupMenuItemFindRelatedMessages == null) {
+			popupMenuItemFindRelatedMessages = new PopupMenuSessionSearch(this);
```

Comment:
Could pass the panel instead of the extension (assuming the search method is dropped).